Formula/vitess.rb

```diff
@@ -0,0 +1,40 @@
+class Vitess < Formula
+  desc "Database clustering system for horizontal scaling of MySQL"
+  homepage "https://vitess.io"
+  url "https://github.com/vitessio/vitess/archive/v8.0.0.tar.gz"
+  sha256 "c47320b9bcb874b1a6dfca78ec677be7c4bb4c7b2a6470df80bd1bc0ad125e92"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
```

Comment:
Go only works using `go@1.13`

Reply:
Seems like it builds fine with the latest version and we don't allow new formula to depend on legacy software.

Reply:
@systay can you explain more? It builds with Go 1.15 and the [local examples](https://vitess.io/docs/get-started/local/) work.

Reply:
@SMillerDev just fyi, go 1.13 isn't legacy, it still receives security updates: https://golang.org/doc/devel/release.html#go1.13.minor

Reply:
Until February, meanwhile there have been 2 new releases. It's not deprecated but it _is_ legacy IMHO.

Reply:
Building might work, but tests do not pass for newer go versions. It's shameful, but still not done. :(

Vitess is only supported on go@1.13 at the moment (I'm one of the [maintainers](https://github.com/vitessio/vitess/blob/master/MAINTAINERS.md) of Vitess)

Reply:
I tend to agree: we can accept this formula when it builds with latest go.